spec/index.html

```diff
@@ -360,12 +364,26 @@ <h3>Triple Terms and Reification</h3>
           aria-label="An RDF graph containing a reifying triple where the triple term corresponds to an asserted triple"></object>
       </a>
       <figcaption id="fig-asserted-triple-term-alt">
-        An <a>RDF graph</a> containing a <a>reifying triple</a> where the <a>triple term</a> corresponds to an <a>asserted triple</a>.
+        An <a>RDF graph</a> containing a <a>triple annotation</a>, which is a <a>reifying triple</a> whose <a>triple term</a> corresponds to an <a>asserted triple</a>.
         The diagram represents the proposition as a fact using the asserted triple, meaning that the relationship holds.
       </figcaption>
     </figure>
 
-    <p>Note that a <a>triple term</a> may also have another <a>triple term</a> as an <a>object</a>.</p>
+    <p class="note">
+      The <a>propositions</a> denoted by <a>triple terms</a> are abstract,
+      logical relationships, and assertions are always made about some
+      <a>reifier</a> thereof. There can be multiple, distinct reifiers
```

Comment:
Maybe 
```suggestion
      A <a>proposition</a> denoted by a <a>triple term</a> is an abstract,
      logical relationship; assertions are always made about some
      <a>reifier</a> thereof. There can be multiple, distinct reifiers
```
because it is one triple term for one proposition.